notebooks/261-fast-segment-anything/261-fast-segment-anything.ipynb

```diff
@@ -18,6 +18,8 @@
     "- [Original Pipeline Inference](#FastSAM-in-Ultralytics)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -q nncf
</pre><p>Maybe move to the top?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Unfortunately, this installation can not be skipped with the help of skip_kernel_extension.py. So I moved the installation to the top of the notebook</p>